content/docs/cml-with-dvc.md

```diff
@@ -15,7 +15,7 @@ on: [push]
 jobs:
   run:
     runs-on: [ubuntu-latest]
-    container: docker://dvcorg/cml-py3:latest
+    container: docker://iterativeai/cml:latest
```

Comment:
either

```suggestion
    container: docker://iterativeai/cml:0-dvc2-base1
```

or

```suggestion
    container: docker://iterativeai/cml:0-dvc2-base1-gpu
```

depending on context. Ditto elsewhere...

Reply:
Should we encourage pinning? Explicit tag names are rather ugly, but give us more freedom to perform breaking changes. Assuming that a project will always be backwards compatible doesn't seem very popular among the succesful ones.